dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/cases/workflow/BaseWorkflowE2ETest.java

```diff
@@ -167,8 +202,15 @@ protected TaskInstanceTab.Row untilTaskInstanceFailed(String workflowName, Strin
         return await()
                 .until(() -> {
                     browser.navigate().refresh();
-                    List<TaskInstanceTab.Row> taskInstances = projectPage
-                            .goToTab(TaskInstanceTab.class)
+
+                    await().ignoreException(NoSuchElementException.class)
+                            .untilAsserted(() -> assertThat(browser)
+                                    .as("task instance is not found")
+                                    .matches(it -> it.findElement(By.className("n-menu-item-content")).isDisplayed()));
+                    TaskInstanceTab taskInstanceTab = projectPage
+                            .goToTab(TaskInstanceTab.class);
+
+                    List<TaskInstanceTab.Row> taskInstances = taskInstanceTab
```

Comment:
Why change this? The current detection process is normal and does not need to be modified.